docs/reference/release-notes/7.7.asciidoc

```diff
@@ -1,22 +1,612 @@
 [[release-notes-7.7.0]]
 == {es} version 7.7.0
 
-coming[7.7.0]
+Also see <<breaking-changes-7.7,Breaking changes in 7.7>>.
 
 [[breaking-7.7.0]]
 [float]
 === Breaking changes
 
+Highlighting::
```

Comment:
Can you expand the explanation and context around these breaking changes abit ? (maybe reach out to the original authors). 

I think people actually read this section and any additional clarity/context here would be very helpful. You can look at 7.6 a guide: https://www.elastic.co/guide/en/elasticsearch/reference/current/breaking-changes-7.6.html

Reply:
oh ... i see this is just the release note doc. We should probably include the breaking changes doc too.  What is here is good assuming it lines up with the breaking changes doc (fine for a different PR)

Reply:
It does kinda line up with https://github.com/elastic/elasticsearch/blob/7.7/docs/reference/migration/migrate_7_7.asciidoc ... but could you use the same headings and order ?

Reply:
Thanks for having a look, @jakelandis!

> It does kinda line up with https://github.com/elastic/elasticsearch/blob/7.7/docs/reference/migration/migrate_7_7.asciidoc ... but could you use the same headings and order ?

IIUC your suggestion: note sure if the two pages are supposed to align perfectly: these release notes are first automatically generated based on the PR's labels and then I manually "tuned" those where labels are lacking or multiple area ones are set. Also, the subsection ordering is same for all sections and entries ordering is decreasing by ID (based on the automatic iteration).
I believe the breaking changes notes are added by PR authors, especially because the release notes might be too succinct.
I'm checking the previous release and that seems to check out (7.6.2 [release](https://www.elastic.co/guide/en/elasticsearch/reference/current/release-notes-7.6.2.html#breaking-7.6.2) vs 7.6 [breaking](https://www.elastic.co/guide/en/elasticsearch/reference/current/breaking-changes-7.6.html) notes).
But I might still be wrong and happy to adjust if needed. Just to make sure, @colings86, wdyt?

Reply:
@bpintea - on second thought, I am in-deed prescribing a change to how we do/have-done things, which is out of scope for this review. Apologies for the noise. These look good as-is.

Reply:
Better safe, so in any case thanks for the check, @jakelandis!